modules/library/render/src/main/java/org/geotools/renderer/lite/StreamingRenderer.java

```diff
@@ -60,7 +59,7 @@
 import org.geotools.data.DataUtilities;
 import org.geotools.data.FeatureSource;
 import org.geotools.data.Query;
-import org.geotools.data.memory.CollectionSource;
```

Comment:
Goodbye CollectionSource - remind me to do a pass through the documentation in case we have any lurking references to it.